pkgs/by-name/ni/nixel/package.nix

```diff
@@ -28,6 +39,8 @@ rustPlatform.buildRustPackage rec {
     version = testers.testVersion { package = nixel; };
   };
 
+  passthru.updateScript = lib.getExe nix-update;
```

Comment:
Use `nix-update-script { }` instead

Reply:
this didnt work on my station when I ran the maintainers/update-script

Reply:
Worked this time :shrug: